tools/src/main/scala/scala/scalanative/codegen/Lower.scala

```diff
@@ -1190,6 +1209,10 @@ object Lower {
   val largeAllocName = extern("scalanative_alloc_large")
   val largeAlloc = Val.Global(largeAllocName, allocSig)
 
+  val zoneAllocSig = Type.Function(Seq(Type.Ptr, Type.Ptr, Type.Size), Type.Ptr)
+  val zoneAllocName = extern("memorypoolzone_alloc")
```

Comment:
Maybe we should prefix the SN internal methods with `scalanative_`, to prevent potential name clashes with user code similarly as it is done `dyndispatchName` few lines below